Tools/environment_install/install-prereqs-ubuntu.sh

```diff
@@ -482,6 +482,44 @@ grep -Fxq "$exportline4" ~/$SHELL_LOGIN 2>/dev/null || {
         echo "Skipping appending CCache to PATH."
     fi
 }
+
+
+UXRCE_GEN_TOOL_NAME="ArduPilot's MicroXRCEDDSGen tool for ROS 2"
+if [[ -z "${DO_UXRCGEN_INSTALL}" ]] && maybe_prompt_user "Install $UXRCE_GEN_TOOL_NAME [N/y]?" ; then
+    DO_UXRCGEN_INSTALL=1
+fi
+
+if [[ $DO_UXRCGEN_INSTALL -eq 1 ]]; then
+    if ! package_is_installed "default-jre"; then
+        heading "Installing default-jre"
+        $APT_GET install default-jre
+        echo "Done!"
+    fi
+
+    if [[ -z "${UXRCE_GEN_ROOT}" ]]; then
+        UXRCE_GEN_ROOT=$OPT/ardupilot_uxrce
+    fi
+
+    heading "Installing $UXRCE_GEN_TOOL_NAME"
+    echo "Downloading from ArduPilot release"
+    sudo mkdir -p $UXRCE_GEN_ROOT
+    cd $UXRCE_GEN_ROOT
```

Comment:
Shouldn't you return to the previous directory after this install step is done?

Reply:
No need. The others don't, they use absolute paths for directory changes.